library/src/main/java/com/mapzen/android/model/DebugFlag.java

```diff
@@ -0,0 +1,14 @@
+package com.mapzen.android.model;
+
+/**
+ * Debug flags available on the map.
+ */
+public enum DebugFlag {
```

Comment:
Given these debug flags are used specifically for Tangram development I'm not sure they belong in the SDK?